napari/layers/points/points_utils.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict, Tuple, Union
+
+import numpy as np
+from vispy.color.colormap import Colormap
+
+
+def dataframe_to_properties(dataframe) -> Dict[str, np.ndarray]:
+    """ Converts a dataframe to Points.properties formatted dictionary
```

Comment:
```suggestion
    """Convert a dataframe to Points.properties formatted dictionary.
```

(Similar formatting suggestions apply to all docstrings. =)